src/config/constants.js

```diff
@@ -113,3 +111,6 @@ export const UTILS_FILES = {
 };
 
 export const CIRCLE_PROGRESS_SIZE = 26;
+
+export const FILE_SIZE_LIMIT_OPTIONS = [500, 1000, 2500, 5000, 10000];
```

Comment:
I put these as tests, let me know which value we should add.

Reply:
Those look good to me. I'm wondering a infinite/no limit option :thinking:

Reply:
It would be nice indeed. Let's create an issue with it as "nice to have".